toolchain/BUILD

```diff
@@ -0,0 +1,90 @@
+load(":cc_toolchain_config.bzl", "cc_toolchain_config")
+package(default_visibility = ["//visibility:public"])
+
+filegroup(name = "empty")
+
+
+cc_toolchain_suite(
+    name = "clang_suite",
+    toolchains = {
+        "aarch64": ":cc-compiler-aarch64",
+        "ppcle": "cc-compiler-ppcle",
+        "s390x": ":cc-compiler-s390x",
+        "x86_32": ":cc-compiler-x86_32",
+        "x86_64": ":cc-compiler-x86_64",
+    },
+)
+
+CC_TOOLCHAIN_CPUS = [
+    "aarch64",
+    "ppcle",
+    "s390x",
+    "x86_32",
+    "x86_64",
+]
+
+[
+    cc_toolchain(
+        name = "cc-compiler-" + cpu,
+        all_files = ":empty",
+        compiler_files = ":empty",
+        dwp_files = ":empty",
+        dynamic_runtime_lib = ":empty",
+        linker_files = ":empty",
+        objcopy_files = ":empty",
+        output_licenses = ["restricted"],
+        static_runtime_lib = ":empty",
+        strip_files = ":empty",
+        toolchain_config = ":" + cpu + "-config",
+        toolchain_identifier = "linux_" + cpu,
+    )
+    for cpu in CC_TOOLCHAIN_CPUS
+]
+
+cc_toolchain_config(
```

Comment:
It'd be good to prefix all the name fields with the OS here as well so they don't collide potentially with other OSes.

Reply:
Done!